Updated Profile (#407)
hms-dbmi/gehlenborglab-website: _members/matthew-scott-tan.md

---
title: Matthew Scott Tan
name_degree: Matthew Scott Tan

photo: matthew-scott-tan.png

job_title: Undergraduate Student in Mathematics and Statistics
role: student # or postdoc, administration, staff, alumni, etc.

services:
  #home: https://member-website.com

>   #github: https://github.com/

Reviewer: Would you like to include your GitHub profile?

```suggestion
  github: https://github.com/mattscott68
```

  #twitter: https://twitter.com/

>   #linked-in: https://www.linkedin.com/in/

Reviewer: or LinkedIn profile?

```suggestion
  linked-in: https://www.linkedin.com/in/matthew-scott-tan/
```

> start: Sep 2023

Reviewer 2: Other members' start dates have used the full month name:

```suggestion
start: September 2023
```

end:
---

> Matthew is an udergraduate stuent concentrating in Mathematics and Statistics at Harvard College. Previously, he was a Specialist (Intelligence) in the Singapore Armed Forces, focusing on imagery and geospatial analysis. At the Gehlenborg lab, he is using computational and mathematical methods to process single-cell resolution omics and imaging data.

Reviewer 2: Minor spelling fix + I believe we've changed the lab name to HIDIVE:

```suggestion
Matthew is an undergraduate student concentrating in Mathematics and Statistics at Harvard College. Previously, he was a Specialist (Intelligence) in the Singapore Armed Forces, focusing on imagery and geospatial analysis. At the HIDIVE lab, he is using computational and mathematical methods to process single-cell resolution omics and imaging data.
```
